### PR TITLE
Adding script for extending bridge config for automated bridge setup/…

### LINKED ISSUE #674: Add script for filling out the additional bridge config fields
Add script for filling out the additional bridge config fields before deleting testnet so that we avoid unnecessary manual work.

### PR DESCRIPTION
**Pull request by Psykepro**
…deploy.

Signed-off-by: Nikolay Nedkov <nikolay.nedkov@limechain.tech>

**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #674

**Special notes for your reviewer**:

**Checklist**
- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: scripts/bridge/setup/extend-config/cmd/extend-bridge-config.go [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+	"fmt"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/limechain/hedera-eth-bridge-validator/app/clients/evm"
+	"github.com/limechain/hedera-eth-bridge-validator/app/clients/evm/contracts/wtoken"
+	mirrorNode "github.com/limechain/hedera-eth-bridge-validator/app/clients/hedera/mirror-node"
+	"github.com/limechain/hedera-eth-bridge-validator/app/domain/client"
+	"github.com/limechain/hedera-eth-bridge-validator/config"
+	"github.com/limechain/hedera-eth-bridge-validator/constants"
+	parser "github.com/limechain/hedera-eth-bridge-validator/scripts/bridge/setup/parser"
```
Comment (svetlio8) [inline scripts/bridge/setup/extend-config/cmd/extend-bridge-config.go:30]: nit alias `parse` -> `/parse`

**Review by svetlio8 — DISMISSED** [review]

**Review by svetlio8 — APPROVED** [review]

File: scripts/bridge/setup/extend-config/cmd/extend-bridge-config.go [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+		return err
+	}
+	assetInfo.Supply = totalSupply.Uint64()
+	if assetInfo.Supply == 0 {
+		assetInfo.Supply = defaultSupply
+	} else if assetInfo.Supply > math.MaxInt64 || assetInfo.Supply < 0 {
+		assetInfo.Supply = math.MaxInt64
+	}
+
+	return err
```
Comment (vectorman1) [inline scripts/bridge/setup/extend-config/cmd/extend-bridge-config.go:248]: nit: `err` here is `nil`, just return `nil`

File: scripts/bridge/setup/extend-config/cmd/extend-bridge-config.go [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+	assetInfo.Name = name
+
+	symbol, err := evmTokenClient.Symbol(&bind.CallOpts{})
+	if err != nil {
+		log.Errorf("EVM with networkId [%d] for Asset [%s], and method Symbol - Error: [%s]", networkId, assetAddress, err)
+		return err
+	}
+	assetInfo.Symbol = symbol
+
+	return err
```
Comment (vectorman1) [inline scripts/bridge/setup/extend-config/cmd/extend-bridge-config.go:271]: nit: `err` here is `nil`, just return `nil`

File: app/services/assets/assets.go [hunk 4]
```diff
@@ -276,9 +276,9 @@ func (a *Service) fetchHederaFungibleAssetInfo(
 		return assetInfo, err
 	}
 
-	assetInfoResponse, e := mirrorNode.GetToken(assetId)
-	if e != nil {
-		a.logger.Errorf("Hedera Mirror Node method GetToken for Asset [%s] - Error: [%s]", assetId, e)
+	assetInfoResponse, err := mirrorNode.GetToken(assetId)
+	if err != nil {
+		a.logger.Errorf("Hedera Mirror Node method GetToken for Asset [%s] - Error: [%s]", assetId, err)
```
Comment (vectorman1) [inline app/services/assets/assets.go:281]: + `return nil, err`

File: app/services/assets/assets.go [hunk 5]
```diff
@@ -276,9 +276,9 @@ func (a *Service) fetchHederaFungibleAssetInfo(
...
 	} else {
```
Comment (vectorman1) [inline app/services/assets/assets.go:282]: else is not necessary

File: app/services/assets/assets.go [hunk 6]
```diff
@@ -290,6 +290,26 @@ func (a *Service) fetchHederaFungibleAssetInfo(
 	return assetInfo, err
```
Comment (vectorman1) [inline app/services/assets/assets.go:290]: nit: `return assetInfo, nil`

**Review by vectorman1 — CHANGES_REQUESTED** [review]

**Review by vectorman1 — APPROVED** [review]

**Pull request merged**